frame/support/src/lib.rs

```diff
@@ -822,6 +822,10 @@ macro_rules! assert_error_encoded_size {
 #[doc(hidden)]
 pub use serde::{Deserialize, Serialize};
 
+#[doc(hidden)]
+#[cfg(not(no_std))]
+pub use macro_magic;
```

Comment:
it supports no_std now as long as you don't have the proc_support feature enabled, btw